SimpactApp/views.py

```diff
@@ -1,3 +1,10 @@
 from django.shortcuts import render
+from django.http import HttpResponse
+
+def home(request):
+    return HttpResponse('<doctype>...')
```

Comment:
I assume these are temporary stubs and you're going to switch to real templates instead.

Reply:
yes! these will be replaced with the html files that we're working on!